src/pyrqlite/connections.py

```diff
@@ -41,7 +41,7 @@ class Connection(object):
     )
 
     def __init__(self, scheme='http', host='localhost', port=4001,
-                 user=None, password=None, connect_timeout=None,
+                 user=None, password=None, timeout=None,
                  detect_types=0, max_redirects=UNLIMITED_REDIRECTS):
```

Comment:
If we remove the `connect_timeout` parameter then also sqlalchemy-rqlite will need to be updated here:

https://github.com/rqlite/sqlalchemy-rqlite/blob/8aa51797ac72523361f3bf531b48f3ea6232d0b3/src/sqlalchemy_rqlite/pyrqlite.py#L76

Reply:
Added backward compat and deprecated it in a8e25175d03290fcba6237de0dbf6c6a1892a436.